AutoMad/madFace.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Oct 17 14:08:59 2024
+
+@author: dragon
+"""
+
+from flask import Flask, render_template, request, Markup
+from madLibFeeder import appleSlices, noPeeling
+import wordGrabber
+import storySaver
+import autoMad
+import soupPourer
+
+
+app = Flask(__name__)
+
+@app.route("/")
+def loadHome():
+    return render_template("home.html",stories=wordGrabber.getStories())
+
+@app.route("/story", methods = ['POST'])
+def story():
+    if request.form.get('stories') is not None:
+        pickedStory = request.form.get('stories')
```

Comment:
```suggestion
    pickedStory = request.form.get('stories')
    if pickedStory is not None:
```
Or something to avoid doing the lookup in two different places